ontotext-yasgui-web-component/src/services/yasqe/yasqe-service.ts

```diff
@@ -145,17 +145,24 @@ export class YasqeService {
 
   /**
    * Initializes the state of infer and same as buttons.
+   *
+   *
+   *
+   *
    * @param yasqe - the yasqe.
+   * @param defaultInfer - default value of infer if not set in <code>yasqe</code>
+   * @param defaultSameAs - default value of sameAs if not set in <code>yasqe</code>
    * @private
    */
   //@ts-ignore
-  private initInferAndSameAsState(yasqe: Yasqe) {
-    if (yasqe.getInfer() === undefined) {
-      yasqe?.setInfer(true);
-    }
-
-    if (yasqe.getSameAs() === undefined) {
-      yasqe?.setSameAs(true);
-    }
+  private initInferAndSameAsState(yasqe: Yasqe, defaultInfer = true, defaultSameAs = true) {
```

Comment:
Defaults should come from our internal model. Right?

Reply:
Yes, your are right. I will remove this defaults.